pkg/kube/services/services_test.go

```diff
@@ -419,3 +420,23 @@ func TestFindURLFromIngress(t *testing.T) {
 		t.Logf("test %s generated URL from Ingress %s", tc.Name, actual)
 	}
 }
+
+func TestFindUrlFromVsIstio(t *testing.T) {
+	var namespace = "default"
+	var name = "vansh"
+	virtualService := &unstructured.Unstructured{}
+	virtualService.SetUnstructuredContent(map[string]interface{}{
```

Comment:
One test case with no hosts will be good (always test more than just the happy path)

Reply:
Hosts is a required field while creating virtual Service and it can't contain 0 elements. So should I do it ?

Reply:
I see, than it's not required.